x-pack/plugins/fleet/public/plugin.ts

```diff
@@ -107,19 +105,34 @@ export class IngestManagerPlugin
 
         return () => {
           unmount();
-          teardownIngestManager(coreStart);
+          teardownFleet(coreStart);
         };
       },
     });
 
+    // BWC < 7.11 redirect /app/ingestManager to /app/fleet
+    core.application.register({
+      id: 'ingestManager',
+      category: DEFAULT_APP_CATEGORIES.management,
+      navLinkStatus: AppNavLinkStatus.hidden,
+      title: i18n.translate('xpack.fleet.oldAppTitle', { defaultMessage: 'Ingest Manager' }),
+      async mount(params: AppMountParameters) {
+        const [coreStart] = await core.getStartServices();
+        coreStart.application.navigateToApp('fleet', {
```

Comment:
Nice.
So this still show a left-nav (kibana) nav item, correct?
should there be a flag (in settings) to possibly remove it? (I'm thinking of non-existing fleet users that may download/install >= 7.11)

Reply:
the navlink is hidden with 
``` 
navLinkStatus: AppNavLinkStatus.hidden,
```